1_Mechanics/02_GMGuide.md

```diff
@@ -5,25 +5,25 @@
 
 <!-- MarkdownTOC -->
 
-- Introduction
-- Getting Started as a Game Master
-    - Making Checks
-    - GM Preparation
-    - Creating Your World
-- Being a Deck of Adventures Game Master
-    - GM Resources
-    - Managing Player Resources
-    - Cards
-    - Resting
-    - Working with Character Vulnerabilities
-- Running a Session with Deck of Adventures
-    - The Target Card System
-    - Successes and Failures
-    - Difficulty Chart
-    - Fate Cards
-    - Skills Breakdown
-    - Contested Checks
-    - Epic Events
+- [Introduction](#Introduction)
```

Comment:
Did you add these heading links by hand? [This package](https://packagecontrol.io/packages/MarkdownTOC) should be able to auto-add them. It's been a while since there were any updates to [the repo](https://github.com/naokazuterada/MarkdownTOC) so there may be some bugs. If it doesn't work for you, it's pretty quick for me to do

Reply:
Yea I hand coded them so we'd have it in there, good to know there's an auto-add tool we can use in the future, thanks!